### PR TITLE
Update JRC Hydro Database version

### PR DESCRIPTION
**Pull request by brynpickering**
From V4 to V7, and update the method extracting data from the zip to be version hash agnostic.

### CODE REVIEW

**Comment by timtroendle** [comment]
Even more relevant: this PR moves from Geth PHS storage capacities to JRC PHS storage capacities.

File: Snakefile [hunk 1]
```diff
@@ -2,7 +2,6 @@ URL_LOAD = "https://data.open-power-system-data.org/time_series/2019-06-05/time_
 URL_POTENTIALS = "https://zenodo.org/record/3533038/files/possibility-for-electricity-autarky.zip"
 URL_CAPACITY_FACTORS = "https://zenodo.org/record/3899687/files/{wildcards.filename}?download=1"
 
-NATIONAL_PHS_STORAGE_CAPACITIES = "data/pumped-hydro/storage-capacities-gwh.csv"
```
Comment (timtroendle) [inline Snakefile:5]: This file should be deleted as it's not necessary anymore.

File: src/hydro.py [hunk 2]
```diff
@@ -14,11 +13,6 @@ def main(path_to_plants, path_to_locations, path_to_phs_storage_capacities, path
                                    fill_storage_capacity=False)
     hdam = capacities_per_location(plants[plants.type == "HDAM"].copy(), locations, tech_type="hdam")
     hphs = capacities_per_location(plants[plants.type == "HPHS"].copy(), locations, tech_type="hphs")
-    hphs["storage_capacity_hphs_MWh"] = scale_phs_storage_capacities(
```
Comment (timtroendle) [inline src/hydro.py:17]: Can you provide some overview over the changes this implies? For example, could you provide national totals in both cases?

**Review by timtroendle — CHANGES_REQUESTED** [review]
The version update is very nice, thanks!

Moving away from Geth data may be fine, but it would be good to understand the consequences. See my comment.

**Comment by brynpickering** [comment]
In terms of changes, they are below. There are some big differences, but I have been reliably informed that the Italian data is much more realistic in JRC than in Geth et al. (and there is an order of magnitude difference between the two). So, it might not be that JRC was vastly overestimating, but that Geth was vastly underestimating...

<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>geth_phs</th>
      <th>jrc_dam</th>
      <th>jrc_phs</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>AL</th>
      <td>NaN</td>
      <td>62267.00</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>AT</th>
      <td>132410.0</td>
      <td>1544800.00</td>
      <td>527400.0</td>
    </tr>
    <tr>
      <th>BA</th>
      <td>NaN</td>
      <td>303.00</td>
      <td>3.4</td>
    </tr>
    <tr>
      <th>BE</th>
      <td>5710.0</td>
      <td>0.00</td>
      <td>5000.0</td>
    </tr>
    <tr>
      <th>BG</th>
      <td>11130.0</td>
      <td>0.00</td>
      <td>0.0</td>
    </tr>
    <tr>
      <th>CH</th>
      <td>311480.0</td>
      <td>1803200.00</td>
      <td>780.0</td>
    </tr>
    <tr>
      <th>CY</th>
      <td>0.0</td>
      <td>NaN</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>CZ</th>
      <td>5720.0</td>
      <td>0.00</td>
      <td>0.0</td>
    </tr>
    <tr>
      <th>DE</th>
      <td>39120.0</td>
      <td>0.00</td>
      <td>35400.0</td>
    </tr>
    <tr>
      <th>DK</th>
      <td>0.0</td>
      <td>NaN</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>EE</th>
      <td>0.0</td>
      <td>NaN</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>ES</th>
      <td>70000.0</td>
      <td>4176442.00</td>
      <td>4261523.0</td>
    </tr>
    <tr>
      <th>FI</th>
      <td>0.0</td>
      <td>0.00</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>FR</th>
      <td>83370.0</td>
      <td>2126400.00</td>
      <td>73776.0</td>
    </tr>
    <tr>
      <th>GB</th>
      <td>26700.0</td>
      <td>0.00</td>
      <td>26700.0</td>
    </tr>
    <tr>
      <th>GR</th>
      <td>4970.0</td>
      <td>1855000.00</td>
      <td>0.0</td>
    </tr>
    <tr>
      <th>HR</th>
      <td>2340.0</td>
      <td>0.00</td>
      <td>14800.0</td>
    </tr>
    <tr>
      <th>HU</th>
      <td>0.0</td>
      <td>0.00</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>IE</th>
      <td>1800.0</td>
      <td>0.00</td>
      <td>0.0</td>
    </tr>
    <tr>
      <th>IT</th>
      <td>68270.0</td>
      <td>2600.00</td>
      <td>398595.0</td>
    </tr>
    <tr>
      <th>LT</th>
      <td>10800.0</td>
      <td>0.00</td>
      <td>0.0</td>
    </tr>
    <tr>
      <th>LU</th>
      <td>4920.0</td>
      <td>NaN</td>
      <td>4478.0</td>
    </tr>
    <tr>
      <th>LV</th>
      <td>0.0</td>
      <td>46330.00</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>ME</th>
      <td>NaN</td>
      <td>2894.00</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>MK</th>
      <td>NaN</td>
      <td>0.00</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>MT</th>
      <td>0.0</td>
      <td>NaN</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>NL</th>
      <td>0.0</td>
      <td>NaN</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>NO</th>
      <td>399390.0</td>
      <td>40547872.95</td>
      <td>4952921.7</td>
    </tr>
    <tr>
      <th>PL</th>
      <td>7960.0</td>
      <td>0.00</td>
      <td>7960.0</td>
    </tr>
    <tr>
      <th>PT</th>
      <td>40770.0</td>
      <td>1334260.00</td>
      <td>181012.0</td>
    </tr>
    <tr>
      <th>RO</th>
      <td>10200.0</td>
      <td>0.00</td>
      <td>10.2</td>
    </tr>
    <tr>
      <th>RS</th>
      <td>NaN</td>
      <td>646350.00</td>
      <td>0.0</td>
    </tr>
    <tr>
      <th>SE</th>
      <td>72120.0</td>
      <td>0.00</td>
      <td>25000.0</td>
    </tr>
    <tr>
      <th>SI</th>
      <td>500.0</td>
      <td>0.00</td>
      <td>0.0</td>
    </tr>
    <tr>
      <th>SK</th>
      <td>3630.0</td>
      <td>0.00</td>
      <td>0.0</td>
    </tr>
  </tbody>
</table>

**Comment by timtroendle** [comment]
Geth is certainly an underestimation as they say themselves they do not cover all plants. The numbers are dramatically different sometimes though, and I personally simply don't know which one's better.

What do you think of keeping Geth data and the scaling mechanism as an option?

**Comment by brynpickering** [comment]
@timtroendle I will update this once #45 is merged in. I will allow one to set a binary config option to scale capacities based on Geth, and default it to False.

**Comment by timtroendle** [comment]
When updating, could you separate this PR into two: one for updating the data (solving #19) and one for making Geth optional? The two do not seem linked to me and the later is a big change.

**Comment by brynpickering** [comment]
> When updating, could you separate this PR into two: one for updating the data (solving #19) and one for making Geth optional? The two do not seem linked to me and the later is a big change.

It would be much easier to keep them in this PR, if that's OK. They are linked in that updating the JRC hydro database is moving closer to the 'reality' of PHS capacity (as that database can be assumed to have only improved), which thus makes the Geth scaling possibly unnecessary, hence that change.

**Comment by brynpickering** [comment]
Ignore that, I'll split them as I ned to rebase onto `develop` branch anyway

**Comment by brynpickering** [comment]
Closed in favour of split into #48 and #49

**Pull request closed without merging**